tests/ConnectionTest.php

```diff
@@ -138,6 +138,20 @@ public function testConnectDispatchEvent(): void
         $conn->connect();
     }
 
+    public function testEventDispatcherIsSetForCustomPlatform(): void
+    {
+        $eventManager = new EventManager();
+
+        $driverMock = $this->createMock(Driver::class);
+
+        $platformMock = $this->createMock(AbstractPlatform::class);
+        $platformMock->expects(self::once())
```

Comment:
Please make it a functional test, if possible. Just the fact that one DBAL component calls a method of another DBAL component doesn't mean anything from the API consumer standpoint.

Reply:
@morozov Done. I have created a functional test.